freqtrade/commands/cli_options.py

```diff
@@ -614,4 +614,36 @@ def __init__(self, *args, **kwargs):
               "that do not contain any parameters."),
         action="store_true",
     ),
+    "analysis_groups": Arg(
+        "--analysis-groups",
+        help=("grouping output - "
+              "0: simple wins/losses by enter tag, "
+              "1: by enter_tag, "
+              "2: by enter_tag and exit_tag, "
+              "3: by pair and enter_tag, "
+              "4: by pair, enter_ and exit_tag (this can get quite large)"),
+        nargs='?',
+        default="0,1,2",
```

Comment:
Looking at usage of this (and the other commands) - i think it would be better to use `nargs='+'`.

That'll allow users to specify `--analysis-groups 1 2 3` (no comma) - and you'll be given a list instead of a string (so no need to split the string anymore).
That would align this better to how other parameters (`trade_ids`, `timeframes` (for downloading) are handling multi-input. 

that's obviously not limited to this command ... 

As the number of selections is limited, maybe using `choices=...` will also help to limit user errors.